Earthfile

```diff
@@ -196,6 +198,21 @@ lint:
     BUILD +shellcheck-lint
     BUILD +yamllint
 
+syft:
+    FROM anchore/syft:latest
+    SAVE ARTIFACT /syft syft
+
+image-sbom:
+    FROM +docker
+    WORKDIR /build
+    COPY +version/VERSION ./
+    ARG VERSION=$(cat VERSION)
+    ARG FLAVOR
+    COPY +syft/syft /usr/bin/syft
+    RUN syft / -o json=sbom.syft.json -o spdx-json=sbom.spdx.json
+    SAVE ARTIFACT /build/sbom.syft.json sbom.syft.json AS LOCAL core-${FLAVOR}-${VERSION}-sbom.syft.json
```

Comment:
shouldnt it be `${VARIANT}-${FLAVOR}-${VERSION}` ? Not sure if we ever change the variant but in case we do that would trickle down to the naming here as well.